ros_installation.md

```diff
@@ -0,0 +1,197 @@
+     * Be sure of using `export GZ_VERSION=garden`
+
+### Gz Fortress
+
+Gz Fortress can be also used with ROS 2 Galactic and ROS 1 Noetic, the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) supports the compilation from
+source.
+
+ * Follow [these instruction to install gz-fortress](https://gazebosim.org/docs/fortress/install_ubuntu#binary-installation-on-ubuntu)
+   from osrfoundation.org repository
+ * Follow the instructions to compile the ros_gz bridge from source in a colcon workspace
+   * [ROS 2 Galactic](https://github.com/gazebosim/ros_gz/tree/galactic#from-source)
+     * Be sure of using `export GZ_VERSION=fortress`
+   * [ROS1 Noetic](https://github.com/gazebosim/ros_gz/tree/noetic#from-source)
+     * Be sure of using `export GZ_VERSION=fortress`
+
+### Gz Edifice
+Gz Edifice can be also used with ROS2 Foxy, ROS2 Rolling and ROS1 Noetic, the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) supports the compilation from
+source.
+
+ * Folow [the instruction to install gz-edifice](https://gazebosim.org/docs/edifice/install_ubuntu#binary-installation-on-ubuntu)
+   from osrfoundation.org repository
+ * Follow the instructions to compile the ros_gz bridge from source in a colcon workspace
+   * [ROS 2 Foxy](https://github.com/gazebosim/ros_gz/tree/foxy#from-source)
+     * Be sure of using `export GZ_VERSION=edifice`
+   * [ROS 2 Rolling](https://github.com/gazebosim/ros_gz/tree/ros2#from-source)
+     * Be sure of using `export GZ_VERSION=edifice`
+   * [ROS1 Noetic](https://github.com/gazebosim/ros_gz/tree/noetic#from-source)
+     * Be sure of using `export GZ_VERSION=edifice`
+
+## Using the latest Gz binary versions
+
+The Gz team usually backport and release new versions of each of the supported
+Gz releases and libraries (i.e: bumping `gz-sim 7.0.0` to gz-sim `7.1.0`). These
+updates are hosted first in the osrfoundation repository. The ROS repository
+syncs from the osrfoundation repository frequently but versions can be
+different.
+
+Getting the latest versions of the Gazebo libraries and simulator is as easy
+as installing the [osrfoundation.org repository](https://gazebosim.org/docs/latest/install_ubuntu_src#install-dependencies)
+together with the ROS repository. Updates should be fully compatible.
+
+## FAQ
+
+#### I am not using ROS at all, which version should I use?
+
+If you don't need ROS support, the recommended version is the latest released
+version that can be [installed using the osrfoundation repo](https://gazebosim.org/docs)
+depending on your platform.
+
+#### I want to use Gz Garden with ROS. Where are the packages?
+
+Unfortunately there are no Gz Garden packages for any ROS distribution. Please
+read how to compile it from source in this document under the section
+"Using an specific Gazebo version with ROS2" >> "Gz Garden"
+
+#### Where I can find the different features implemented on each Gazebo version?
+
+Some notes are regularly posted on the [Gazebo community
```

Comment:
I think https://gazebosim.org/docs/garden/release-features would be a good link to put here.

Reply:
Indeed, that is the best option. [572f672](https://github.com/gazebosim/docs/pull/339/commits/572f672e2208102c39713e1be07577407418d124)